teachopencadd/talktorials/T016_protein_ligand_interactions/talktorial.ipynb

```diff
@@ -4,58 +4,83 @@
    "cell_type": "markdown",
```

Comment:
<blockquote><a href="https://plip.biotec.tu-dresden.de/plip-web/plip/index" target="_blank">PLIP</a>&nbsp;is one of the most popular thanks to its publicly available webserver and free-to-use Python library</blockquote><p>-&gt; a popular (not sure if 'most' is necessarily correct)</p><p> </p><p>I would prefer if instead of quoting the paper here, we would have a bit more (maybe bullet point like) explanation in our own words, listing the interactions and the general idea of how they are calculated</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>